env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
     log.setDefaultLevel('debug');
   } else {
     log.setDefaultLevel('info');
   }
 
   return {
+    legacyMode: process.env.LEGACY_MODE || false,
```

Comment:
We could at least call "legacyMode" something more descriptive like "0x-mode" (can't work out the right cases or whatever).

The whole thing about shipping versions together sounds good to me, but it does dictate our deployment infrastructure a little, because then we need to ensure they're upgraded together, which lends itself towards bundling as a gardener module IMO.